.github/workflows/test.yml

```diff
@@ -51,16 +51,6 @@ jobs:
           manifest-dir: ${{ github.workspace }}/.github/manifest
           github-binarycache: true
 
-      - name: Cargo Build Windows
-        if: matrix.os == 'windows-latest'
-        run: |-
-          $env:SODIUM_LIB_DIR="$(pwd)\vcpkg\packages\libsodium_x64-windows-release\lib"
-          cargo build --all-targets
-
-      - name: Cargo Build
-        if: matrix.os != 'windows-latest'
-        run: cargo build --all-targets
-
```

Comment:
Different builds no longer required, I suppose?

Reply:
cargo build has been moved into the Makefile, so the `make unit` below now covers this step.